content/en/docs/languages/java/intro.md

```diff
@@ -0,0 +1,175 @@
+
+Click the link in the "Managed Dependencies" column to see a list of the
+artifacts managed by the BOM.
+
+| Description                                                                                  | Repository                           | Group ID                           | Artifact ID                               | Current Version                            | Managed Dependencies                                      |
+| -------------------------------------------------------------------------------------------- | ------------------------------------ | ---------------------------------- | ----------------------------------------- | ------------------------------------------ | --------------------------------------------------------- |
+| Stable core API and SDK artifacts                                                            | `opentelemetry-java`                 | `io.opentelemetry`                 | `opentelemetry-bom`                       | `{{% param vers.otel %}}`                  | [latest pom.xml][opentelemetry-bom]                       |
+| Experimental core API and SDK artifacts, including all of `opentelemetry-bom`                | `opentelemetry-java`                 | `io.opentelemetry`                 | `opentelemetry-bom-alpha`                 | `{{% param vers.otel %}}-alpha`            | [latest pom.xml][opentelemetry-bom-alpha]                 |
+| Stable instrumentation artifacts, including all of `opentelemetry-bom`                       | `opentelemetry-java-instrumentation` | `io.opentelemetry.instrumentation` | `opentelemetry-instrumentation-bom`       | `{{% param vers.instrumentation %}}`       | [latest pom.xml][opentelemetry-instrumentation-bom]       |
+| Experimental instrumentation artifacts, including all of `opentelemetry-instrumentation-bom` | `opentelemetry-java-instrumentation` | `io.opentelemetry.instrumentation` | `opentelemetry-instrumentation-bom-alpha` | `{{% param vers.instrumentation %}}-alpha` | [latest pom.xml][opentelemetry-instrumentation-alpha-bom] |
+
+Add a BOM dependency with Maven:
+
+```xml
+<project>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>{{bomGroupId}}</groupId>
+        <artifactId>{{bomArtifactId}}</artifactId>
+        <version>{{bomVersion}}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <!-- Add a dependency on an artifact whose version is managed by the bom -->
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+    </dependency>
+  </dependencies>
+</project>
+```
+
+Add a BOM dependency with Gradle:
+
+```kotlin
+dependencies {
+  implementation(platform("{{bomGroupId}}:{{bomArtifactId}}:{{bomVersion}}"))
+  // Add a dependency on an artifact whose version is managed by the bom
+  implementation("io.opentelemetry:opentelemetry-api")
+}
+```
```

Comment:
We could also consider using a tabpane for these. My code suggestion gets all messed up since it has code backticks in it, but: 

```
Add a BOM dependency::

{{< tabpane text=true >}} {{% tab "Maven" %}}

```xml
<project>
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>{{bomGroupId}}</groupId>
        <artifactId>{{bomArtifactId}}</artifactId>
        <version>{{bomVersion}}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
  <!-- Add a dependency on an artifact whose version is managed by the bom -->
  <dependencies>
    <dependency>
      <groupId>io.opentelemetry</groupId>
      <artifactId>opentelemetry-api</artifactId>
    </dependency>
  </dependencies>
</project>
```

{{% /tab %}} {{% tab "Gradle" %}}

```kotlin
dependencies {
  implementation(platform("{{bomGroupId}}:{{bomArtifactId}}:{{bomVersion}}"))
  // Add a dependency on an artifact whose version is managed by the bom
  implementation("io.opentelemetry:opentelemetry-api")
}
```

{{% /tab %}} {{< /tabpane >}}
```

Reply:
@jaydeluca if you want to make suggestions with back ticks in it, you can use three tildes (`~`) instead of the backticks:

~~~
```
this works
```
~~~